docs/openapi.yaml

```diff
@@ -1347,6 +1349,22 @@ components:
       required: false
       schema:
         type: string
+    viewingScope:
```

Comment:
unsure if we need a new parameter? As I understood it from Terje we should change the existing one for contributor, and add the excludeSubunits